karma.conf.js

```diff
@@ -15,6 +15,7 @@ module.exports = function(config) {
 
     // list of files / patterns to load in the browser
     files: [
+      'node_modules/babel-polyfill/dist/polyfill.min.js',
```

Comment:
I haven't tested it in Internet Explorer but probably should.  I included this polyfill here because I started running the e2e specs in Phantomjs which also is missing a lot of the same stuff.

I created an issue here: https://github.com/Autodesk/molecule-3d-for-react/issues/13